src/NUnit.Xml.TestLogger.TestAdapter/NUnit.Xml.TestLogger.TestAdapter.csproj

```diff
@@ -24,8 +24,11 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="15.5.0" />
-    <PackageReference Include="Spekt.TestLogger" Version="$(TestLoggerVersion)" />
     <PackageReference Include="System.ValueTuple" Version="4.3.0" />
   </ItemGroup>
 
+    <ItemGroup>
```

Comment:
We may need to undo this for the CI to pass.

Reply:
Yeah, I will. Its just for my dev right now